### DISCUSSION TITLE
Click to go to comment

### DISCUSSION THREAD

**Discussion by theblockchainheadhunter** [sonigy/YCS#9] [Ideas]
It would be great if we can search/filter the comments and then click on them to see the whole theat.
I did a video with a use case
https://share.getcloudapp.com/lluoWPDn

Thanks for the plugin Eugene

Javier from Spain  !!!

**Comment by sonigy** [comment]
Hi @javiergomez698,
You can interact with a comment by clicking on its date.

How to like, reply to a comment?
In the search results, click on the date (like, "2 months ago") of the comment and will open a new window with an active comment or reply under the video, where you can do any action.

**Comment by theblockchainheadhunter** [comment]
Thanks so much for the answer Eugene.
If it possible for you to make the comment clikable people will find it much easier.
Just a user experience recomendation
Thanks again

**Comment by eedo31** [comment]
Yeah, being able to interact with the comment by tapping on it, or even better, directly without having to open it, would be great. I discovered the date thing by chance; it's not intuitive at all.
Tbf, it's still, easily, one of the best extensions for savvy YouTube viewers, but this could definitely be improved.
